src/sections/Header.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
```

Comment:
This component should be in the `layouts` directory

Reply:
Moved to the Layouts